static/usage/breadcrumbs/collapsing-items/popover-on-click/angular/angular_ts.md

```diff
@@ -0,0 +1,21 @@
+```ts
+import { Component, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+})
+export class AppComponent {
+  @ViewChild('popover') popover;
+  constructor() {}
```

Comment:
~ We should probably remove the constructor when it isn't being used with providers. Angular style conventions recommend having public variables before the constructor.